src/words.js

```diff
@@ -0,0 +1,6 @@
+const words = [
+  'Кіт','Песик','Лисиця','Ведмідь','Вовк',
+  'Панда'
```

Comment:
Use objects with English for names instead of just Ukrainian strings and then use these names to get correct translations

Reply:
everything is here src\data\wordObjects.js